tools/cli/test/e2e/cli/split_test.go

```diff
@@ -113,7 +122,8 @@ func ValidateVersionedSpec(t *testing.T, correctSpecPath, generatedSpecPath stri
 	t.Helper()
 	correctSpec := newOpenAPISpec(t, correctSpecPath)
 	generatedSpec := newOpenAPISpec(t, generatedSpecPath)
-	d, err := diff.Get(diff.NewConfig(), correctSpec, generatedSpec)
+	examples := []string{"examples"}
+	d, err := diff.Get(diff.NewConfig().WithExcludeElements(examples), correctSpec, generatedSpec)
```

Comment:
removing examples because our generated spec is matching with the expected examples after we split, but the old ones are not, so I'll ignore for now.

Reply:
could you add a comment with the explanation so that we don't forget about it?

Reply:
yes will add in the follow-up PR!